python/private/parse_requirements_add_dists.bzl

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A simple helper to populate the distribution attributes for each entry returned
+by the parse_requirements function.
+
+TODO @aignas 2024-05-23: The name is up for bikeshedding. For the time being I
+am keeping it together with parse_requirements.bzl.
+"""
+
+load(":whl_target_platforms.bzl", "select_whls")
+
+def parse_requirements_add_dists(requirements_by_platform, index_urls, python_version):
```

Comment:
Would this be better put in parse_requirements.bzl? It just seems closely coupled to it.